src/env.cc

```diff
@@ -617,7 +617,7 @@ Local<Value> Environment::GetNow() {
   CHECK_GE(now, timer_base());
   now -= timer_base();
   if (now <= 0xffffffff)
```

Comment:
Should this have an `U` suffix…?

Reply:
It shouldn't matter in this case, right? I thought they were equivalent since this won't fit into an `int` and the next type is `unsigned int`.

Reply:
Okay, I looked it up – without the suffix it is still signed, but it’s a `long int` in that case, so we’re probably fine here. :)